### PR TITLE
Bug 1179891 - Fix for failure in test_ftu_skip_tour.py, with message: No languages listed on screen

### PR DESCRIPTION
**Pull request by mwargers**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=64c0e01c28d4b2c0f7bae42c663a1c0b4e66289c

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e3cf70ae01f746247bf9ca6abb1768b3d8851f12

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/ftu/test_ftu_skip_tour.py [hunk 1]
```diff
@@ -80,11 +81,22 @@ def test_ftu_skip_tour(self):
...
-        self.wait_for_condition(
-            lambda m: not self.data_layer.get_setting('geolocation.enabled'),
-            message='Geolocation was not disabled by the FTU app')
+        # Verify the "geolocation" option matches expected initial state
+        # There's a very small lag on this after the dialog appears.
+        Wait(self.marionette).until(
+            lambda m: self.ftu.is_geolocation_enabled == geolocation_default,
+            message="Geolocation state should match initial internal state")
+
+        # Verify that internal state matches UI state
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/ftu/test_ftu_skip_tour.py]: We might not need to check the internal state now that we write acceptance tests. See: https://wiki.mozilla.org/B2G/QA/Automation/UI/Integration_suite_criteria

Comment (npark-mozilla) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/ftu/test_ftu_skip_tour.py]: But unless there is another test that can do this check currently, perhaps we should wait until we deprecate this?

Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/ftu/test_ftu_skip_tour.py]: Yeah, that was the conclusion in our daily standup as well. So I'll check this in and we can always change it when we do have integration tests on device.

**Pull request merged**